templates/create.jinja2

```diff
@@ -0,0 +1,20 @@
+{% extends "base.jinja2" %}
+{% block body %}
+{% if request.authenticated_userid %}
+  <aside>
```

Comment:
given that the entire purpose of the "create" page is to create a new entry, is the form really part of an "aside"?  Is it not, rather, a central element on the page?  

The `<aside />` element is intended as a semantic container that indicates the materials contained in it is of lesser or peripheral importance to the meaning of the page.  Is that appropriate here?